<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
<suite thread-count="10" name="Failed suite [Suite]">
  <test name="Inquiry Internal Functionalities(failed)">
    <classes>
      <class name="testScript.TC_InquiryInternalFunctionalities">
        <methods>
          <include name="TC_VideoCall"/>
          <include name="TC_UnArchiveInquiry"/>
          <include name="TC_RemoveParticipant"/>
          <include name="beforeTest"/>
          <include name="afterTest"/>
          <include name="TC_ArchiveInquiry"/>
          <include name="TC_DeleteNotes"/>
          <include name="TC_EditInquiryDetails"/>
        </methods>
      </class> <!-- testScript.TC_InquiryInternalFunctionalities -->
      <class name="testScript.TC_NewInquirywithInvalidData">
        <methods>
          <include name="afterTest"/>
          <include name="beforeTest"/>
        </methods>
      </class> <!-- testScript.TC_NewInquirywithInvalidData -->
      <class name="testScript.TC_NewInquiry">
        <methods>
          <include name="beforeTest"/>
          <include name="afterTest"/>
        </methods>
      </class> <!-- testScript.TC_NewInquiry -->
    </classes>
  </test> <!-- Inquiry Internal Functionalities(failed) -->
</suite> <!-- Failed suite [Suite] -->
